consensus/poet/core/sawtooth_poet/poet_consensus/poet_settings_view.py

```diff
@@ -16,12 +16,12 @@
 import math
```

Comment:
Having "atomic" commits would be my preference as well.  I would like a policy established on which way we go on this so that these type of moves/renames are done the same way all of the time.  When I moved the PoET consensus code out from the validator package into its own package, I originally followed the approach that moving + fixing were in a single commit so that the commit in isolation didn't break anything.  However, at the time it was requested that I change the commit to split it into a change/rename commit and then a fixup commit.